lib/engine/auto_router.rb

```diff
@@ -155,13 +161,19 @@ def compute(corporation, **opts)
         puts ' Node timeout reached' if node_abort
       end
 
+      @flash&.call('Auto route path walk failed to complete (PATH TIMEOUT)') if path_walk_timed_out
```

Comment:
does this actually work given that the calls are blocking?

Reply:
It displays, but the combo loop is normally so fast I can't tell if the flash blocks until function ends.

Reply:
i'd put it at the end anyways

Reply:
I just verified that it won't flash until the function ends.  If both timeouts occur, I'm not sure if user will see both flashes.

Reply:
just combine them into one

Reply:
ok